tests/bench/refusalbench_common.py

```
@@ -0,0 +1,541 @@
+        quality_score: float | None = None
+        if quality_score_raw is not None and quality_score_raw != "null":
+            try:
+                quality_score = float(quality_score_raw)
+            except (TypeError, ValueError):
+                quality_score = None
+        explanation: str = judgment.get("explanation", "")
+
+        return predicted_type, quality_score, explanation
+
+    except Exception as e:
+        logger.exception("Error in judge_refusal_response: %s", e)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)